source/standards/running-services-as-containers.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: How to run your service as a container
+
+expires: 2018-05-02
+---
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+If your web application is running on [GOV.UK Platform as a Service (PaaS)][] you should follow the [recommended hosting guidance][].
+
+If your web application is running on PaaS you should use:
+
+* [Docker][] to [build and deploy images][], as it’s widely used across GDS and industry
+* [Dockerhub][] or [Amazon EC2 Container Registry][] to host your images following the [PaaS responsibility model][]
+* the recommended [monitoring and logging solution][]
+
+You can also run your service using bare metal Docker on an [Amazon Elastic Compute Cloud (Amazon EC2)][] instance if:
+
+* service uptime is not a priority
+* zero downtime deployments are not required
+* it can be managed by configuration management
```

Comment:
we could put a link here to the configuration management guidance once it's also published